web/client/utils/ogc/Filter/fromObject.js

```diff
@@ -26,16 +28,41 @@ const operators = {
  * const ogcFilter = toOgcFiler(filterObject);
  * // ogcFilter --> "<ogc:PropertyIsEqualTo><ogc:PropertyName>property</ogc:PropertyName><ogc:Literal>value</ogc:Literal></ogc:PropertyIsEqualTo>"
  */
-const fromObject = (filterBuilder = {}) => ({type, filters = [], value, property, lowerBoundary, upperBoundary }) => {
+const fromObject = (filterBuilder = {}) => ({type, filters = [], args = [], name, value, ...rest }) => {
+    if (type === "literal") {
+        return filterBuilder.literal(value);
+    }
+    if (type === "property") {
+        return filterBuilder.propertyName(name);
+    }
     if (includes(logical, type)) {
         return filterBuilder[type](
             ...filters.map(fromObject(filterBuilder))
         );
     }
     if (includes(cql, type)) {
-        return "";
+        return ""; // TODO: implement in filterBuilder as empty filter
```

Comment:
what about this?

Reply:
Include and Exclude are only part of CQL/ECQL of geoserver. They do not have a proper equivalent in OGC filters.
This is a suggestion for a future implementation of emptyFilter in filterBuilder, equivalent to the "include" cql filter